dev/a11y_assessments/web/index.html

```diff
@@ -35,12 +35,6 @@
 
   <title>a11y_assessments</title>
   <link rel="manifest" href="manifest.json">
-
-  <script>
```

Comment:
do you know what this does?

Reply:
This doesn't do anything any more. It's gone. The app was generating warnings because it still used the defunct API.